### PR TITLE
Forged Alliance Forever Client

### PR DESCRIPTION
**Pull request by Eonfge**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### Draft
Hey all. Over the years, I've packaged quite a few applications but this time I'll limit myself to a draft for now. This application is currently not working and I want to help Upstream in making their client suitable for use with Flathub.

https://github.com/FAForever/downlords-faf-client/issues/2416

### Known issues
- Doesn't run
- Even if it would run, it might not be properly configured yet
- Some incorrect .desktop entries
- Test file `run.sh` should not be part of the submission

### CODE REVIEW

**Comment by Eonfge** [comment]
bot, build

**Comment by Eonfge** [comment]
bot, build com.faforever.Client

**Comment by Eonfge** [comment]
bot, build com.faforever.Client

**Comment by Eonfge** [comment]
bot, build com.faforever.Client

File: com.faforever.Client.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+app-id: com.faforever.Client
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: /app/bin/faf-client
```
Comment (hfiguiere) [inline com.faforever.Client.yaml:5]: it's never an absolute path.

File: com.faforever.Client.desktop [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Type=Application
+Name=FAForever
+Comment=Supreme Commander Forged Alliance game lobby
+Exec=/app/bin/faf-client
```
Comment (hfiguiere) [inline com.faforever.Client.desktop:5]: Exec in .desktop is not an absolute path either.

**Comment by Eonfge** [comment]
bot, build com.faforever.Client

**Pull request closed without merging**